build-system/compile/bundles.config.extensions.json

```diff
@@ -157,10 +157,37 @@
   {"name": "amp-facebook-page", "version": "0.1", "latestVersion": "0.1"},
   {
     "name": "amp-fit-text",
-    "version": ["0.1", "1.0"],
+    "version": "0.1",
     "latestVersion": "0.1",
     "options": {"hasCss": true}
   },
+  {
+    "name": "amp-fit-text",
+    "version": "1.0",
+    "latestVersion": "0.1",
+    "options": {
+      "hasCss": true,
+      "binaries": [
```

Comment:
Ok for now, but this is going to be quite a mess after awhile and would be good place these configs into the folder of each extension. E.g. `extensions/amp-fit-text/1.0/bundle.config.json`?

Reply:
Yah, I'm going to provide a standard "preact" and "react" configuration, and we'll just provide `{ preact: entryPoint, react: entryPoint }` configuration. That should take care of all the duplication comments, while leaving an abstract "binaries" configuration for more advanced cases.